client/components/mma/identity/emailAndMarketing/EmailAndMarketing.stories.tsx

```diff
@@ -1,5 +1,5 @@
 import type { Meta, StoryObj } from '@storybook/react';
```

Comment:
is this file just formatting changes?

Reply:
if there are so many formatting issues it might be worth running a format on the whole codebase and putting it in as a single PR (and you can ask github to ignore it in the "blame" somehow)